txtl-liposome_water-in-oil.md

```diff
@@ -0,0 +1,217 @@
+# TX-TL Liposome Using Water-in-Oil Emulsion
+
+Version 0.1.0, 8 August 2018
+
+Contributors:
+Aaron Engelhart (University of Minnesota),
+Jan Gregrowicz (Caltech),
+Joseph Heili (University of Minnesota),
+Zoila Jurado (Caltech),
+Neha Kama (Northwestern),
+Akshay Maheshwari (Stanford),
+Richard Murray (Caltech),
+Milena Popovic (Blue Marble Space),
+Kazhito Tabata (University of Tokyo),
+Paola Torre (University of Pennsylvania).
+
+## Overview and Materials
+
+This protocol describes how to create liposomes that contain TX-TL
+inside of a lipid-based container. The liposomes created by these
+protocols are between 1 and 10 um in diameter and should be usable
+with an cell-free protein expression system.
+
+The preparation of liposomes using w/o emulsions as template requires
+four steps:
+
+### **Step 0**: Deposition of a thin lipid film in a glass vial
```

Comment:
I think we can get rid of the emphasis on Step 0 since this is a level 3 heading.

Reply:
Would we do this before completing the pull request or after and start a new pull request?

Reply:
Normally what happens is that the reviewers (curators) make suggestions for changes that should be completed before the pull request is finalized.  So I would make these changes in my version of the code (murrayrm/liposome-kit) and then it will show up here as a response to the review.

Reply:
Probably you forgot me as contributor - but if I can't be there for any reason,  a simple acknowledgement is okay of course. Please decide as you wish.

Reply:
Probably you forgot my name as contributor. But if for any reason it cannot be there, an acknowledgement is still fine.